src/app/static/src/app/components/downloadResults/UploadModal.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+    <div id="dialog">
+        <modal :open="open">
+            <h4 v-translate="'uploadFileToAdr'"></h4>
+            <div class="container">
+                <div id="dataset-id" class="mt-4">
+                    <span v-translate="'uploadFileDataset'"></span>
+                    <span>{{ dataset }}</span></div>
+                <div id="instructions" class="mt-3" v-translate="'uploadFileInstruction'"></div>
+                <div id="output-file-id" class="mt-3" v-for="uploadFile in uploadFiles" :key="uploadFile.index">
+                    <div class="mt-3 form-check">
+                        <input class="form-check-input"
+                               type="checkbox"
+                               :value="uploadFile.index"
```

Comment:
This should be the key of the map entry (e.g. `outputSummary`), not the index of the value - see comment on test below